testah-junit/src/main/java/org/testah/driver/web/browser/FirefoxBrowser.java

```diff
@@ -68,13 +74,14 @@ public DesiredCapabilities createCapabilities() {
         capabilities.setCapability("nativeEvents", false);
         capabilities.setCapability(FirefoxDriver.PROFILE, profile);
         capabilities.setCapability("elementScrollBehavior", 1);
-        if (null != TS.params().getWebDriver_firefoxDriverBinary() && TS.params().getWebDriver_firefoxDriverBinary().length() > 0) {
+        if (null != TS.params().getWebDriver_firefoxDriverBinary()
```

Comment:
https://commons.apache.org/proper/commons-collections/javadocs/api-3.2.2/org/apache/commons/collections/CollectionUtils.html#isEmpty(java.util.Collection) seems cleaner

Reply:
Think you mean string utils?  Ya much cleaner, will update